bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Core/SystemUsers/SystemUser.cs

```diff
@@ -21,35 +21,45 @@ public class SystemUser
     public string Id { get; set; } = string.Empty;
 
     /// <summary>
-    /// The Title and Description are strings set by the end-user in the Frontend.
-    /// In the db this field is required.
+    /// The Title is by default the same as the System's Display Name
     /// </summary>
     [JsonPropertyName("integrationTitle")]
     public string IntegrationTitle { get; set; }
 
     /// <summary>
-    /// For off the shelf systems.
-    /// Should probably be human readable (instead of a GUID) but unique string without whitespace
-    /// The "real" Authentication Component should validate that the SystemName is unique
-    /// Retrieved from the SystemRegister, the full CRUD Api is in a different service
-    /// In the db this field is required, but if we use this model as a DTO, we allow null
+    /// Identifier for off the shelf systems, registered in the SystemRegister db.
+    /// Should be human readable (instead of a GUID) and unique string without whitespace.                
+    /// </summary>
+    [JsonPropertyName("systemId")]
+    public string SystemId { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Identifier for off the shelf systems, registered in the SystemRegister db.
+    /// Should be human readable (instead of a GUID) and unique string without whitespace.                
+    /// To be deprecated, use the systemId field going forward, it contains the same value.
     /// </summary>
     [JsonPropertyName("productName")]
-    public string ProductName { get; set; } = string.Empty;
+    public string ProductName { get { return SystemId; } set { } } 
```

Comment:
why do we need to retain this in BFF?

Reply:
1. because I haven't checked if the React frontend uses it 
2. to be symmetrical with the auth comp
3. we will remove it when we no longer need it for maskinporten